### PR TITLE
184258400 More Missing Operands

### PR DESCRIPTION
**Pull request by tealefristoe**
This PR adds more cases for the missing operand warning message, as well as new cypress tests for them.

### CODE REVIEW

**Comment by tealefristoe** [comment]
I'm not sure why this is complaining about test coverage--I explicitly added tests to cover all of the new cases. Maybe the problem is that the tests have been added in a loop instead of hard coded directly?

**Review by emcelroy — APPROVED** [review]
Looks good to me! I think you're right that Codecov isn't "smart" enough to count your new tests toward the coverage percentage. If you're sure they're being run and passing, I think it's fine to ignore that.

File: cypress/integration/error-messages.test.ts [hunk 1]
```diff
@@ -31,15 +31,15 @@ context("Test error messages", () => {
...
-    });
+    const testIncomplete = (expression: string) => {
+      it (`renders incomplete for "${expression}"`, () => {
+        enterExpression(expression);
+        computedNode().find(".error-short").should("contain.html", "Um, still working?");
+      });
+    };
+    const incompleteStrings = ["+ a + b", "+a", "*a+b", "a+*b", "/a+b", "a+/b", "^a+b",
+      "a+^b", "(a+b", "a(+b", "a+(b", "a+b)"];
+    incompleteStrings.forEach(expression => testIncomplete(expression));
```
Comment (kswenson) [inline cypress/integration/error-messages.test.ts:42]: I think the `it()` needs to be at the top level and the looping should happen within it.

Comment (tealefristoe) [reply cypress/integration/error-messages.test.ts]: Thanks, that fixed the issue.

**Pull request merged**